.circleci/config.yml

```diff
@@ -838,16 +872,10 @@ parameters:
     run_commit_main:
         default: true
         type: boolean
-    run_deliver_beta_administration:
+    run_deliver_beta_backend_administration:
         default: false
         type: boolean
-    run_deliver_beta_backend:
-        default: false
-        type: boolean
-    run_deliver_production_administration:
-        default: false
-        type: boolean
-    run_deliver_production_backend:
+    run_deliver_production_backend_administration:
```

Comment:
maybe we just call it backend?

Reply:
I like this naming, as it is more clear what is happening. I prefer having a few extra chars instead of not understanding at the first look what is going on.